testproject/django_file_form_example/forms.py

```diff
@@ -54,6 +54,8 @@ def save(self):
         )
 
         for f in self.cleaned_data['input_file']:
+            if hasattr(f, 'placeholder'):
```

Comment:
This is how backend handles placeholder during save. This should be removed later or added to a separate example.